plugins/com.google.cloud.tools.eclipse.util/src/com/google/cloud/tools/eclipse/util/Xslt.java

```diff
@@ -42,7 +42,9 @@ public static void transformInPlace(IFile file, URL xslt)
     try (InputStream in = file.getContents();
         InputStream stylesheetStream = xslt.openStream();
         InputStream resultStream = applyXslt(in, stylesheetStream)) {
-      file.setContents(resultStream, IFile.FORCE, null /* monitor */);
+        boolean force = true;
+        boolean keepHistory = true;
+        file.setContents(resultStream, force, keepHistory, null /* monitor */);
```

Comment:
So, accordingly to the Javadoc, seems like `KEEP_HISTORY` is recommended almost always:

> Specifying <code>KEEP_HISTORY</code> is recommended except in circumstances where past states of the files are of no conceivable interest to the user.

In that regard, it'd be worth to see if there is any other `setContents()` usage in our code without the `KEEP_HISTORY` flag.